docs/source/index.rst

```diff
@@ -1,20 +1,83 @@
-.. docker-compose documentation master file, created by
-   sphinx-quickstart on Wed Apr 10 14:02:48 2019.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+DeviceManager
```

Comment:
I don't understand why this section is here.

Reply:
I think it was a mistake

Reply:
I will remove the file